deploy/kubernetes/helm-charts/altinn-runtime/values.yaml

```diff
@@ -7,24 +7,24 @@ image:
 
 environmentVariables:
   development:
-    - name: GiteaEndpoint
-      value: dev.altinn.studio
-    - name: GiteaLoginEndpoint
-      value: https://dev.altinn.studio/user/login
-    - name: GiteaApiEndpoint
-      value: https://dev.altinn.studio/api/v1
     - name: ASPNETCORE_ENVIRONMENT
       value: Development
     - name: ASPNETCORE_URLS
       value: http://+:5005
+    - name: ServiceRepositorySettings__ApiEndPointHost
+      value: altinn-repositories
```

Comment:
Will these altinn-repositories hostnames work?

Reply:
@matsgm  the hostname is used to set the domain of a cookie that is set. Are you thinking it should be changed to dev.altinn.studio?